src/sentry/snuba/models.py

```diff
@@ -38,6 +38,22 @@ class Meta:
         unique_together = (("query_subscription", "environment"),)
 
 
+class SnubaQuery(Model):
```

Comment:
Does this new model need to be wired up to deletions?

Reply:
Hmm, not sure. The only reference from this -> an org is the `Environment`. We might just be able to rely on Django removing these rows when the environment is deleted.